frontend/src/pages/Admin/Components/Resource/Resource.jsx

```diff
@@ -1,9 +1,69 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { END_POINT } from "../../../../config/api";
+import axios from "axios";
+import "./resource.module.scss";
+import style from "./resource.module.scss";
 
 export function Resource() {
+  const [resourses, setResources] = useState([]);
+  useEffect(() => {
+    const token = localStorage.getItem("token");
+    const fetchResource = async () => {
+      try {
+        const data = await axios.get(`${END_POINT}/resources/getresources`, {
+          headers: { Authorization: `Bearer ${token}` },
+        });
+        const resource = await data.data;
+        setResources(resource);
+      } catch (err) {
+        console.log("There was some error fetching resource page");
```

Comment:
If any error occurs, can we please show a toast msg of errors instead?

Reply:
ok on it